src/PrestaShopBundle/Resources/config/services/adapter/product.yml

```diff
@@ -405,6 +405,7 @@ services:
     arguments:
       $connection: '@doctrine.dbal.default_connection'
       $dbPrefix: '%database_prefix%'
+      $entityNameDuplicator: '@PrestaShop\PrestaShop\Adapter\Util\Entity\EntityNameDuplicator'
```

Comment:
I think the autowire would have found it automatically since there's only one instance, but not blocking